### ISSUE TITLE
predictor matrix with consecutively numbered variables

### ISSUE THREAD

**Issue by tbuet** [stefvanbuuren/mice#165]
Hi,

there might be a small bug in the following situation (e.g. when imputing longitudinal data using a so-called "wide approach")
```{r}
library(mice)

data <- data.frame(y.1 = c(1,2,3), b = c(3,7,4), y.10 = c(1,2,6))
pred <- make.predictorMatrix(data)
pred
```
same with
```{r}
ini <- mice(data, maxit = 0)
ini$predictorMatrix
```
y.10 isn't identified as predictor for y.1 by default.

**Comment by gerkovink** [comment]
Hi @tbuet, 

Thanks for letting us know. This issue is now addressed in #166. 

All the best, 

Gerko

**Comment by stefvanbuuren** [comment]
Thanks. Now merged.
